apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -402,203 +394,109 @@ const AvailabilityPage = ({ profile, eventType }: Props) => {
       <div>
         <main
           className={classNames(
-            shouldAlignCentrally ? "mx-auto" : "",
-            isEmbed
-              ? classNames(maxWidth)
-              : classNames("transition-max-width mx-auto my-0 duration-500 ease-in-out md:my-24", maxWidth)
+            "flex flex-col",
+            shouldAlignCentrally ? "items-center" : "items-start",
+            !isEmbed && classNames("mx-auto my-0 ease-in-out md:my-24")
           )}>
-          <div
-            style={availabilityDatePickerEmbedStyles}
-            className={classNames(
-              isBackgroundTransparent
-                ? ""
-                : "dark:bg-darkgray-100 sm:dark:border-darkgray-300 bg-white pb-4 md:pb-0",
-              "border-bookinglightest overflow-hidden rounded-md md:border",
-              isEmbed ? "mx-auto" : maxWidth
-            )}>
-            {/* mobile: details */}
```

Comment:
<img width="423" alt="Screenshot 2022-10-10 at 4 44 39 PM" src="https://user-images.githubusercontent.com/1780212/194853942-60440ae5-e14f-4ab0-9ca3-046ce632d312.png">
Back button wasn't there earlier on mobile(should be a bug), but it is showing now. 

Location of Back button should improve for mobile.Seems ill placed.

Reply:
Attention to detail, nice!! The breakpoint was wrong indeed, updated it from sm to md like it should've been. Thanks! 🙏